bot/handlers/bot_message_handler.py

```diff
@@ -84,83 +76,94 @@ def get_parent_class_name(self) -> str:
         return self.__class__.__bases__[0].__name__
 
     async def get_response(self, key: str, args: Optional[List[str]] = None, as_parent: Optional[bool] = False) -> str:
-        if as_parent:
-            return await get_response(key=key, handler_name=self.get_parent_class_name(), args=args)
-        return await get_response(key=key, handler_name=self.get_action_name(), args=args)
+        name = self.get_parent_class_name() if as_parent else self.get_action_name()
+        return await get_response(key=key, handler_name=name, args=args)
 
     @abstractmethod
     def get_commands(self) -> List[str]:
         pass
 
     @abstractmethod
-    async def _do_handle(self, message: Message) -> None:
+    async def _do_handle(self) -> None:
         pass
 
-    @staticmethod
-    async def _answer_markdown(message: Message, text: str) -> None:
-        await message.answer(text, parse_mode="Markdown", reply_to_message_id=message.message_id,disable_notification=True)
+    async def _answer_markdown(self, text: str) -> None:
+        await self._responder.send_markdown(text)
 
-    @staticmethod
-    async def _answer(message: Message, text: str) -> None:
-        await message.answer(text, reply_to_message_id=message.message_id,disable_notification=True)
+    async def _answer(self, text: str) -> None:
+        await self._responder.send_text(text)
 
-    @staticmethod
-    async def _answer_photo(message: Message, image_bytes: bytes, image_path: Path, caption: str) -> None:
-        await message.answer_photo(
-            photo=BufferedInputFile(image_bytes, str(image_path)),
-            caption=caption,
-            show_caption_above_media=True,
-            parse_mode="Markdown",
-            reply_to_message_id=message.message_id,
-            disable_notification=True,
-        )
-
-    async def _answer_video(self, message: Message, file_path: Path) -> None:
-        file_size = file_path.stat().st_size / (1024 * 1024)  # size in MB
+    async def _answer_photo(self, image_bytes: bytes, image_path: Path, caption: str) -> None:
+        await self._responder.send_photo(image_bytes, image_path, caption)
+
+    async def _answer_video(self, file_path: Path) -> None:
+        file_size = file_path.stat().st_size / (1024 * 1024)  # MB
         await self._log_system_message(logging.INFO, get_clip_size_log_message(file_path, file_size))
 
         if file_size > settings.TELEGRAM_FILE_SIZE_LIMIT_MB:
```

Comment:
po co komentarz jak mozna wcisnac w nazwe

```suggestion
        file_size_mb = file_path.stat().st_size / (1024 * 1024)
        await self._log_system_message(logging.INFO, get_clip_size_log_message(file_path, file_size_mb))

        if file_size_mb > settings.TELEGRAM_FILE_SIZE_LIMIT_MB:
```

Reply:
sprytnie